apps/outline/config.json

```diff
@@ -0,0 +1,83 @@
+{
+    "name": "Outline",
+    "id": "outline",
+    "available": true,
+    "short_desc": "A home for all your docs",
+    "author": "outline",
+    "port": 8404,
+    "categories": ["utilities"],
+    "description": "Outline is a knowledge base designed for teams. It's goals are to be fast, intuitive and support many integrations.",
+    "tipi_version": 1,
+    "version": "0.75.2",
+    "source": "https://github.com/outline/outline",
+    "website": "https://getoutline.com",
+    "exposable": true,
+    "force_expose":true,
+    "form_fields": [
+      {
+        "type": "random",
+        "label": "Outline secret key",
+        "min": 64,
+        "env_variable": "OUTLINE_SECRET_KEY"
+      },
+      {
+        "type": "random",
+        "label": "Outline utils secret",
+        "min": 32,
+        "env_variable": "OUTLINE_UTILS_SECRET"
+      },
+      {
+        "type": "random",
+        "label": "Minio access key",
+        "min": 32,
+        "env_variable": "MINIO_ACCESS_KEY"
+      },
+      {
+        "type": "random",
+        "label": "Minio secret key",
+        "min": 32,
+        "env_variable": "MINIO_SECRET_KEY"
+      },
+      {
+        "type": "random",
+        "label": "OIDC client ID",
+        "min": 32,
+        "env_variable": "OIDC_CLID"
+      },
+      {
+        "type": "random",
+        "label": "OIDC client secret",
+        "min": 32,
+        "env_variable": "OIDC_CLKEY"
+      },
+      {
+        "type": "random",
+        "label": "Database password",
+        "min": 32,
+        "env_variable": "PG_PASSWORD"
+      },
+      {
+        "type": "random",
+        "label": "OIDC secret",
+        "min": 32,
+        "env_variable": "OIDC_SECRET"
+      },
+      {
+        "type": "text",
+        "label": "Username",
+        "max": 50,
+        "min": 3,
+        "required": true,
+        "env_variable": "OUTLINE_USER"
+      },
+      {
+        "type": "password",
+        "label": "Password",
+        "min": 12,
+        "required": true,
+        "env_variable": "OUTLINE_PASSWORD"
+      }
+    ],
```

Comment:
It would be better to add `OUTLINE_` prefix to all of the env vars so for example `OUTLINE_OIDC_SECRET`.

Reply:
The variables are local, so they will not interfere with others. I have each one according to the service that uses it.

Reply:
Yeah of course but it's kind of the "runtipi-apps-scheme", basically the way we like everything to be configured. Also when someone checks the app.env file for an x reason and just seed OIDC_SERVER chances are he will not understand what it is about, but if you include the OUTLINE_ thing he will say ohhh yeahh it's outline related not tipi. I am not forcing you to change it but if you can it would be nice.

Reply:
Fixed